joomla/joomla-cms
# Language Override space

When you have a language override that is in both locations then it is displayed as site /administrator

This extra space is added because the markup is on multiple lines.

### To Test
Go to language overrides and select a language-admin pair
Create a new override and check the box for both locations
Save and check the list of overrides

### Before
![image](https://user-images.githubusercontent.com/1296369/187164981-5b02b9f7-2722-4947-8b4a-cee6e7a971db.png)

### After
![image](https://user-images.githubusercontent.com/1296369/187164891-00131e00-46fe-4f0c-b65f-f87264b54b83.png)
